poetry/installation/executor.py

```diff
@@ -611,9 +612,13 @@ def _download_link(self, operation, link):
 
         return archive
 
-    def _download_archive(self, operation, link):  # type: (Operation, Link) -> Path
+    def _download_archive(self, operation, link, repository):  # type: (Operation, Link, LegacyRepository) -> Path
         response = self._authenticator.request(
-            "get", link.url, stream=True, io=self._sections.get(id(operation), self._io)
+            "get",
+            link.url,
+            stream=True,
+            io=self._sections.get(id(operation), self._io),
+            verify=getattr(repository, 'cert'),
```

Comment:
Only `LegacyRepository` has the `cert` attribute, should I have an `isinstance()` check or is `getattr()` good enough?